presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -646,3 +646,50 @@ The following query will fail with the error ``Column 'name' is ambiguous``::
     SELECT name
     FROM nation
     CROSS JOIN region;
+
+
+Subquery expressions
+--------------------
+
+Subquery is an expression which is composed of a SQL query. The subquery is called correlated when refers to variables from the surrounding query. Such variables will act as constants during any one evaluation of the subquery.
+
+.. note:: The subquery expression will generally only be executed far enough to determine infromation required by surrounding query, not all the way to completion (see ref:`exists` and ref:`in_predicate`). It is unwise to write a subquery that has any side effects; whether the side effects occur or not may be difficult to predict.
+
+.. note:: Support for correlated subqueries is experimental and not all form of such expressions are elible for Presto to be exuected.
```

Comment:
@kokosing : I think what you really mean is "limited", not "experimental". I.e., not all types of subqueries work but those that work are working correctly and users should be worried about the correctness of the results etc. Experimental is more like Beta, not guaranteed to work correctly.